rviz_plugins/eStop_plugin/.gitignore

```diff
@@ -0,0 +1,3 @@
+/bin/
+/build/
+/lib/
```

Comment:
@alchaussee We should figure out how to merge these into the main .gitignore. Not a huge fan of cascading gitignores if it can be avoided.